kinto2xml/verifier.py

```diff
@@ -0,0 +1,107 @@
+"""Parse and normalize two XML files and then generate a diff of both."""
+import argparse
+import json
+import logging
+import os
+import requests
+import subprocess
+import six
+import sys
+import tempfile
+import xmltodict
+
+logger = logging.getLogger('xml-verifier')
+
+
+def get_unique_id(*fields):
+    def wraps(d):
```

Comment:
Can you add some comments here, I don't understand this function (its title versus its body :))